metadata-ingestion/source_docs/athena.md

```diff
@@ -0,0 +1,57 @@
+# Athena
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[athena]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Database and table metadata
```

Comment:
What does this mean? It just extracts all tabs and ingests as Datasets to DataHub